earth_enterprise/src/fusion/config/geconfigureassetroot.cpp

```diff
@@ -148,7 +155,13 @@ int main(int argc, char *argv[]) {
 
     // Make sure I'm root, everything is shut down,
     // and switch to fusion user
-    std::string thishost = ValidateHostReadyForConfig();
+    std::string thishost;
+    if (listvolumes) {
+      AssertRunningAsRoot();
+      thishost = GetAndValidateHostname();
+    } else {
+      thishost = ValidateHostReadyForConfig();
+    }
     printf("Switching to (%s, %s)\n", username.c_str(), groupname.c_str());
```

Comment:
@tst-ppenev Unchanged code - out of scope for this ticket. Feel free to raise another ticket.